### PR TITLE
Set up todo

### PR DESCRIPTION
**Pull request by Kandy-Peter**
This pull request has the following contents and steps:

- I installed Webpack and its dependencies
- create the todo list template using HTML/CSS/javascript
- display todo list tasks from javascript using array and object  in javascript
- install linters and fix errors.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+for (let i = 0; i <= todoObjectArray.length; i += 1) {
+  const todoObject = todoObjectArray[i];
+
+  taskList.innerHTML += `<ul class="todoTask">
+                            <li><input type="checkbox" id="checkbox" name="checkbox"></li>        
+                            <li class="todoName">${todoObject.title}</li>        
+                            <li><img src="./images/dots.png" alt="icon"></li>       
+  </ul>`;
+}
```
Comment (leolpaz) [inline src/index.js:31]: - [ ] To comply with the requirement screenshotted below kindly add something either here or above it that sorts the array before rendering them on the page so they show in order based on their index(id) number
image

**Review by leolpaz — CHANGES_REQUESTED** [review]
Hi @Kandy-Peter , 

You've done an amazing job so far, I'd like to highlight the following:
- Good job setting up webpack

Some issues need to be resolved before your approval, but you're almost there!




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  },
+  {
+    id: 1,
+    title: 'walking during the night',
+    completed: false,
+  },
+];
+
+const taskList = document.getElementById('task-list');
+const newTodoArray = todoObjectArray;
```
Comment (SajjadAhmad14) [inline src/index.js:22]: Please make sure the tasks are rendered on screen in the order of their index values as it's asked to implement by previous code reviewer.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Kandy, 
 ### You have been doing a tremendous job so far :ok_hand:

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by ShoiraTa — APPROVED** [review]
### **STATUS: APPROVED  ✔️** 
------
Hi @Kandy-Peter ,  

Your project is complete!  it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 
You have done a great job!



Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**